vs_utils/features/tests/nnscore_test.py

```diff
@@ -43,130 +43,262 @@ def setUp(self):
 
     # TODO(rbharath): This load sequence is getting unwieldy. Refactor.
 
-    ### PRGR is from the DUD-E collection
-    prgr_receptor = PDB()
-    prgr_pdb = os.path.join(data_dir(), "prgr_hyd.pdb")
-    prgr_pdbqt = os.path.join(data_dir(), "prgr_hyd.pdbqt")
-    prgr_receptor.load_from_files(prgr_pdb, prgr_pdbqt)
-    # This compound is CHEMBL1164248
-    prgr_active = PDB()
-    prgr_active_pdb = os.path.join(data_dir(), "prgr_active0_hyd.pdb")
-    prgr_active_pdbqt = os.path.join(data_dir(), "prgr_active0_hyd.pdbqt")
-    prgr_active.load_from_files(prgr_active_pdb, prgr_active_pdbqt)
-
-    ### c-Abl is taken from the Autodock Vina examples
-    cabl_receptor = PDB()
-    cabl_receptor_pdb = os.path.join(data_dir(), "c-Abl_hyd.pdb")
-    cabl_receptor_pdbqt = os.path.join(data_dir(), "c-Abl_hyd.pdbqt")
-    cabl_receptor.load_from_files(cabl_receptor_pdb,
-        cabl_receptor_pdbqt)
-    # This compound is imatinib
-    cabl_active = PDB()
-    cabl_active_pdb = os.path.join(data_dir(), "imatinib_hyd.pdb")
-    cabl_active_pdbqt = os.path.join(data_dir(), "imatinib_hyd.pdbqt")
-    cabl_active.load_from_files(cabl_active_pdb,
-        cabl_active_pdbqt)
-
-    ### 1zea comes from PDBBind-CN
-    # Python complains about variables starting with numbers, so put an
-    # underscore in front of everything.
-    _1zea_protein = PDB()
-    _1zea_protein_pdb = os.path.join(data_dir(), "1zea_protein_hyd.pdb")
-    _1zea_protein_pdbqt = os.path.join(data_dir(), "1zea_protein_hyd.pdbqt")
-    _1zea_protein.load_from_files(_1zea_protein_pdb, _1zea_protein_pdbqt)
+    #### PRGR is from the DUD-E collection
```

Comment:
Can you you make these extra tests runnable by flag rather than just commenting them out?